Content.Client/Light/Components/LightBehaviourComponent.cs

```diff
@@ -52,7 +52,7 @@ public abstract class LightBehaviourAnimationTrack : AnimationTrackProperty
         [ViewVariables] protected float MaxTime { get; set; }
 
         private float _maxTime = default;
-        private EntityUid _parent = default!;
+        private EntityUid? _parent = null!;
```

Comment:
think this one shouldn't be nullable give how it gets assigned in init with a non-nullable value and never gets changed or checked against default/null

Reply:
ah yeah, missed the `!` here and didn't think about it too hard.